helper/pgpkeys/flag.go

```diff
@@ -21,7 +21,36 @@ func (p *PubKeyFilesFlag) Set(value string) error {
 	if len(*p) > 0 {
 		return errors.New("pgp-keys can only be specified once")
 	}
+
+	// First, resolve all the keybase entries...do it in one go so we only
+	// round trip to the API once, then store locally
+	keybaseMap := map[string]string{}
+	keybaseUsers := []string{}
+	for _, keyfile := range strings.Split(value, ",") {
+		if strings.HasPrefix(keyfile, "keybase:") {
+			keybaseUsers = append(keybaseUsers, strings.TrimPrefix(keyfile, "keybase:"))
+		}
+	}
+	keybaseKeys, err := FetchKeybasePubkeys(keybaseUsers)
```

Comment:
I'll see you and raise you one: I'll have the helper method check the prefix and ignore any that don't match, so you don't need to do the initial filter either.